apis/beacon/blocks/block.v2.yaml

```diff
@@ -13,6 +13,13 @@ get:
   responses:
     "200":
       description: "Successful response"
+      headers:
+        Eth-Consensus-Version:
+          description: Required in ssz responses so client can deserialize returned data.
```

Comment:
Do we want to state that this is required in all responses, to allow more efficient decoding of JSON streams?

Reply:
Should I just extract the header to "constant" and put it on every endpoint?

Reply:
Sorry when I said "all responses" I meant for both JSON and SSZ responses for the particular endpoints for which it is relevant.

We can always add it to additional endpoints if it is required in future, but not sure we should add it as a blanket header for all endpoints.